Pipfile

```diff
@@ -5,11 +5,13 @@ verify_ssl = true
 
 [dev-packages]
 ansible-lint = "~=3.4.23"
-molecule = "~=2.19.0"
+molecule = "*"
 docker-py = "*"
 
 [packages]
 ansible = "~=2.6.7"
+molecule = "*"
+docker-py = "*"
```

Comment:
This should stay in dev dependencies, since they're not needed on the jumpboxes. I think I forgot to update the README that dependencies should be installed with `pipenv install --dev` for development.

Reply:
will change